std_msgs/Int8 EPS_En
std_msgs/Int8 EPS_Override_Ignore
std_msgs/Float32 EPS_Speed 
std_msgs/Int8 ACC_En
std_msgs/Int8 AEB_En
std_msgs/Int8 Turn_Signal
std_msgs/Float32 AEB_decel_value  # UInt8 is used, considering the range and scaling factor
std_msgs/Int8 Aliv_Cnt
std_msgs/Float32 EPS_Cmd
std_msgs/Float32 ACC_Cmd